blobstore/cli/clustermgr/config_background.go

```diff
@@ -0,0 +1,184 @@
+)
+
+func addCmdConfigBackground(cmd *grumble.Command) {
+	backgroundCommand := &grumble.Command{
+		Name:     "background",
+		Help:     "background task switch",
+		LongHelp: "Background task switch control for clustermgr, currently supported: " + BackgroundTaskTypeString,
+	}
+	cmd.AddCommand(backgroundCommand)
+
+	backgroundCommand.AddCommand(&grumble.Command{
+		Name:     "status",
+		Help:     "show status of a background task switch",
+		LongHelp: "Show status of a specific background task type, currently supported: " + BackgroundTaskTypeString,
+		Args: func(a *grumble.Args) {
+			a.String("task", "background task type")
+		},
+		Run: cmdListBackgroundStatus,
+		Flags: func(f *grumble.Flags) {
+			flags.VerboseRegister(f)
+			clusterFlags(f)
+		},
+	})
+	backgroundCommand.AddCommand(&grumble.Command{
+		Name:     "enable",
+		Help:     "enable background task",
+		LongHelp: "Enable a specific background task type, currently supported: " + BackgroundTaskTypeString,
+		Args: func(a *grumble.Args) {
+			a.String("task", "background task type")
+		},
+		Flags: clusterFlags,
+		Run: func(c *grumble.Context) error {
+			return cmdEnableDisableBackgroundTask(c, true)
+		},
+	})
+	backgroundCommand.AddCommand(&grumble.Command{
+		Name:     "disable",
+		Help:     "disable background task",
+		LongHelp: "Disable a specific background task type, currently supported: " + BackgroundTaskTypeString,
+		Args: func(a *grumble.Args) {
+			a.String("task", "background task type")
+		},
+		Flags: clusterFlags,
+		Run: func(c *grumble.Context) error {
+			return cmdEnableDisableBackgroundTask(c, false)
+		},
+	})
+}
+
+func cmdEnableDisableBackgroundTask(c *grumble.Context, en bool) error {
+	key := c.Args.String("task")
+
+	found := false
+	for _, kind := range BackgroundTaskTypes {
+		if kind == key {
+			found = true
+			break
+		}
+	}
+
+	if !found {
+		return fmt.Errorf("Unsupported background task type: %s", key)
+	}
+
+	value := "true"
+	act, acted := "enable", "enabled"
+	if !en {
+		value = "false"
+		act, acted = "disable", "disabled"
+	}
+
+	cli := newCMClient(c.Flags)
+	ctx := common.CmdContext()
+	oldV, err := cli.GetConfig(ctx, key)
+	if err != nil {
+		if rpc.DetectStatusCode(err) != http.StatusNotFound &&
+			!common.Confirm("To "+act+" background task of : "+common.Loaded.Sprint(key)+" ?") {
+			return err
+		}
+	}
+
+	if oldV == value {
+		fmt.Printf("Background task of type `%s` has already been %s.", key, acted)
+		return nil
+	}
+
+	if "" == oldV {
+		oldV = notSet
+	}
+
+	fmt.Printf("Current status of background task of `%s`:\n", key)
+	showConfig(key, oldV, true)
+
+	if common.Confirm(fmt.Sprintf(
+		"To %s background task `%s` from `%s` --> `%s` ?", act, common.Loaded.Sprint(key),
+		common.Danger.Sprint(oldV), common.Normal.Sprint(value))) {
+		err := cli.SetConfig(ctx, &clustermgr.ConfigSetArgs{
+			Key:   key,
+			Value: value,
+		})
+		if nil != err {
```

Comment:
The usual usage is `if err!=nil`

Reply:
Sorry, it's resulted from my c++ experience. Solved already. Please check the latest commit.